app/assets/stylesheets/dashboard.scss

```diff
@@ -297,3 +302,21 @@ p.debate-status {
     }
   }
 }
+
+@-webkit-keyframes slideIn {
+  0% {
+    transform: translateX(-900px);
+  }
+  25% {
+    transform: translateX(0);
+  }
+}
+
+@-webkit-keyframes slideOut {
```

Comment:
wouldn't this work without prefixing? (or should we also add prefix for firefox?)